Documentation-Requirements.md

```diff
@@ -12,6 +13,26 @@ Containers](https://github.com/kata-containers) project.
 - All documents are expected to be written in [GitHub Flavored Markdown](https://github.github.com/gfm) format.
 - All documents should have the `.md` file extension.
 
+# Notes
+
+Important information that is not part of the main document flow should be
+added as a Note in bold with all content contained within a quoted block:
```

Comment:
s/quoted block/block quote/ to align with the [markdown spec wording](https://github.github.com/gfm/#block-quotes) (I had to go look that up, as it didn't quite feel right, but I didn't know what the correct phrase was going to be either ;-)

Reply:
Oddly, that was the original wording I used, so I've reverted to that ;)